composer.json

```diff
@@ -188,7 +188,7 @@
 			"rm .eslintrc.json",
 			"rm .gitattributes",
 			"rm .gitignore",
-			"rm .php-cs-fixer.php",
+			"rm Build/php-cs-fixer/.php-cs-fixer.php",
```

Comment:
Please let's keep this list sorted. Also, I suggest we do a `rm -rf Build` instead.